web-ng/src/app/components/layer-dialog/layer-dialog.component.css

```diff
@@ -47,3 +47,54 @@
   margin-right: 0;
   font-weight: bold;
 }
+.link-add-question {
+  color: green;
+  cursor: pointer;
+}
+
+.add-question {
+  padding: 20px;
+  border: 1px solid lightgray;
+  border-radius: 10px;
+  margin-top: 20px;
+}
+
+.plus-icon {
```

Comment:
Is there an Angular Material component we can use to render this styling instead?

Reply:
I tried finding mat-icon but seems like there is +1 but no isolated +